BLSQ/iaso
# POLIO-1686: bOPV hide column

Related JIRA tickets : POLIO-1686

## Self proofreading checklist


## Doc
N/A

## Changes

- Add conditions to remove unusable vials/doses if vaccine is bOPV
- Update "broken" incident report to remove "unusable vials" fieldd if vaccine is bOPV

## How to test
Just go to Vaccine Module > Stock management and switch vaccines to see interface changes

## Print screen / video

https://github.com/user-attachments/assets/c389e9ba-c34e-406d-974c-237effd98cff
